lib/sqlalchemy/testing/reflection_fixture.py

```diff
@@ -0,0 +1,201 @@
+            is_true("computed" not in data[key])
+        compData = data["computed_col"]
+        is_true("computed" in compData)
+        is_true("sqltext" in compData["computed"])
+        actual = compData["computed"]["sqltext"]
+
+        eq_(self.to_sqltext("normal", "+", "42"), actual)
+        eq_("persisted" in compData["computed"], self.return_persisted)
+        if self.return_persisted:
+            eq_(compData["computed"]["persisted"], self.default_persisted)
+
+    def check_column(self, data, column, sqltext, persisted):
+        is_true("computed" in data[column])
+        compData = data[column]["computed"]
+        actual = compData["sqltext"]
+        eq_(sqltext, actual)
+        if self.return_persisted:
+            is_true("persisted" in compData)
+            is_(compData["persisted"], persisted)
+
+    def test_get_column_returns_persisted(self):
+        insp = inspect(config.db)
+
+        cols = insp.get_columns("computed_column_table")
+        data = {c["name"]: c for c in cols}
+
+        self.check_column(
+            data,
+            "computed_no_flag",
+            self.to_sqltext("normal", "+", "42"),
+            self.default_persisted,
+        )
+        if self.support_virtual:
+            self.check_column(
+                data,
+                "computed_virtual",
+                self.to_sqltext("normal", "+", "2"),
+                False,
+            )
+        if self.support_stored:
+            self.check_column(
+                data,
+                "computed_stored",
+                self.to_sqltext("normal", "-", "42"),
+                True,
+            )
+
+    def test_get_column_returns_persisted_with_schama(self):
+        insp = inspect(config.db)
+
+        cols = insp.get_columns(
+            "computed_column_table", schema=config.test_schema
+        )
+        data = {c["name"]: c for c in cols}
+
+        self.check_column(
+            data,
+            "computed_no_flag",
+            self.to_sqltext("normal", "/", "42"),
+            self.default_persisted,
+        )
+        if self.support_virtual:
+            self.check_column(
+                data,
+                "computed_virtual",
+                self.to_sqltext("normal", "/", "2"),
+                False,
+            )
+        if self.support_stored:
+            self.check_column(
+                data,
+                "computed_stored",
+                self.to_sqltext("normal", "*", "42"),
+                True,
+            )
+
+    def check_table_column(self, table, name, text, persisted):
+        is_true(name in table.columns)
+        col = table.columns[name]
+        is_not_(col.computed, None)
+        is_true(isinstance(col.computed, Computed))
+
+        eq_(str(col.computed.sqltext), text)
+        if self.return_persisted:
+            eq_(col.computed.persisted, persisted)
+        else:
+            is_(col.computed.persisted, None)
+
+    def test_table_reflection(self):
```

Comment:
technically I think I'm trying to have all the reflection tests in suite/ only use the inspector, and then testing that the inspector can work these things into a Table with Computed objects is in test/engine/test_reflection.py.

Reply:
I placed it here since the I needed stored/virtual support. I'll try to refactor them in the requirements

Reply:
yeah, im not sure that is going to look much better but I want to try to keep the number of approaches in the test suite to a minimum, even though there are many at this point within the "suite" part of things, that's what we export to third party libs so it should try to be very consistent.